app/pb_kits/playbook/pb_textarea/textarea.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Playbook
+  module PbTextarea
+    class Textarea
+      include Playbook::Props
+
+      partial "pb_textarea/textarea"
+
+      prop :label
```

Comment:
```suggestion
      prop :object
      prop : name
      prop :label
```

Reply:
These props need to be added as mentioned in `text_area.html.erb`

Reply:
@thestephenmarshall Why is object a prop?